EIPS/eip-1679.md

```diff
@@ -26,6 +26,10 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 
 - TBD
 
+### Rejected EIPs
+
```

Comment:
Does 1559 make this list? It sound more like a default "no". Then a flat out rejection? Should there be a place for things that have been tabled and are unlikely to make it without considerable work, and things that have been rejected and aren't going to be considered?

Reply:
Or is it better to make a note of this similar to how Progpow has a sub-note?

Reply:
Well this is just adding the new templates and not documenting any decisions.